asyncapi/json-schemas/sensors/telemetry/state-of-charge/state-of-charge-02.example.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "State of charge",
+  "payload": {
+    "eventTimestamp": "2022-05-09T13:27:59.624Z",
+    "traceId": "3841a268-0c03-4588-b476-211be0f26a0d",
+    "messageNumber": 1,
+    "id": "01000005",
+    "payloads": [
+      {
+        "name": "State of charge",
+        "sensorId": "XB7F-9T2M-L4FC-3R8Q",
+        "value": 20.3
+      }
+    ]
+  }
+}
```

Comment:
Er dette helt likt eksempel 1 ?

Reply:
Oppdaterte de to andre eksemplene til å bruke subId